src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -505,6 +505,13 @@ void GeneralPage::initLayout(GeneralPageView &layout)
                      "message to help better tell them apart.")
         ->addTo(layout);
 
+    SettingWidget::checkbox("Reduce opacity of past messages",
```

Comment:
i think we call them 'historical' in the UI?

Reply:
we call them `history` even tho `historical` is the correct word

Reply:
So true I changed it to "message history" which is the name some other settings also use